build.sbt

```diff
@@ -1,27 +1,42 @@
 name := "magellan"
 
-version := "1.0.3"
+version := "1.0.3-SNAPSHOT"
 
 organization := "harsha2010"
 
 scalaVersion := "2.10.4"
 
-parallelExecution in Test := false
+crossScalaVersions := Seq("2.10.5", "2.11.7")
+
+sparkVersion := "1.4.1"
```

Comment:
Note that bumping the build Spark version from 1.4.0 to 1.4.1 is necessary for the 2.11 tests to work in Travis, for some reason.